conf/views/replaceConfirmation.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "includes/layout.njk" %}
+
+{% from "macros/title.njk"                         import title %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block pageTitle %}
+  {{ title(messages("replaceConfirmation.title")) }}
+
+{% endblock %}
+
+{% block mainContent %}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-panel__title">
+          {{ govukPanel({
+            titleText: messages("replaceConfirmation.heading")
+          }) }}
+        </h1>
+        <p class="govuk-body">
+            {{ messages("replaceConfirmation.p1") }}
+        </p>
+        <h2 class="govuk-heading-m">{{ messages("replaceConfirmation.heading.two") }}</h2>
+         <p class="govuk-body">{{ messages("replaceConfirmation.p2") }}</p>
+        <ul class="govuk-list">
+          <li>{{ messages("replaceConfirmation.p3") }}
+             <a class="disclose-link" href={{discloseBorderArrangements}}>{{messages("replaceConfirmation.p4")}}</a>
+           </li>
+          <li> <a href="/">{{ messages("replaceConfirmation.p5") }}</a> {{ messages("replaceConfirmation.p6") }}</li>
+        </ul>
+      </div>
+    <div class="govuk-grid-column-two-thirds">
+          <p><a href="/">{{ messages("replaceConfirmation.p7") }}</a></p>
+        </div>
```

Comment:
This is constant on all pages and can be added to layout.njk so it is displayed on all pages. This can be done on another ticket.